bangazonapi/views/profile.py

```diff
@@ -274,12 +273,43 @@ def favoritesellers(self, request):
                 }
             ]
         """
-        customer = Customer.objects.get(user=request.auth.user)
-        favorites = Favorite.objects.filter(customer=customer)
+        if request.method == "GET":
+
+            customer = Customer.objects.get(user=request.auth.user)
```

Comment:
Not a blocker, but `customer` could go outside of your `if` blocks since it runs either way. That, and without an auth'd customer you may want to return a `bad request`.